File: app/src/main/java/com/oddlyspaced/calci/mathtype/enums/MTStringInputBehavior.java
package com.oddlyspaced.calci.mathtype.enums;
/* loaded from: classes.dex */
public enum MTStringInputBehavior {
    None,
    IfEmptyGrabSelectedRange,
    IfEmptyGrabRangeLeftOfCursor,
    IfEmptyGrabRangeRightOfCursor,
    IfNotFilledGrabCursor,
    KeepContentsWhenBackspacingParent,
    SelectContentsWhenBackspacingParent,
    SelectContentsWhenBackspacingParentUnlessEqualToImplicitGrab,
    GrabCursorWhenBackspacingParentInsteadOfDelete,
    SelectParentWhenBackspacingAtIndex0,
    DeleteParentWhenBackspacingAtIndex0,
    DeleteParentWhenLeftEmpty
}

File: app/src/main/java/com/oddlyspaced/calci/mathtype/enums/MTStringInputBehavior.kt
package com.oddlyspaced.calci.mathtype.enums

enum class MTStringInputBehavior {
    None,
    IfEmptyGrabSelectedRange,
    IfEmptyGrabRangeLeftOfCursor,
    IfEmptyGrabRangeRightOfCursor,
    IfNotFilledGrabCursor,
    KeepContentsWhenBackspacingParent,
    SelectContentsWhenBackspacingParent,
    SelectContentsWhenBackspacingParentUnlessEqualToImplicitGrab,
    GrabCursorWhenBackspacingParentInsteadOfDelete,
    SelectParentWhenBackspacingAtIndex0,
    DeleteParentWhenBackspacingAtIndex0,
    DeleteParentWhenLeftEmpty,
}